src/main/resources/fxml/Settings.fxml

```diff
@@ -54,7 +54,21 @@
                     <children>
                         <TextField fx:id="webAPIKeyField" prefHeight="31.0" prefWidth="400.0" />
                     </children>
-                </HBox> <Label text="Dynamic Link Domain">
+                </HBox>
+            <Label text="Database URL">
```

Comment:
Database URL could mean a number of things in this context. Pick a different label, maybe Firebase Realtime Database URL or Firebase Database URL.